llvm/tools/llvm-exegesis/lib/SerialSnippetGenerator.cpp

```diff
@@ -52,13 +52,16 @@ computeAliasingInstructions(const LLVMState &State, const Instruction *Instr,
       continue;
     if (OtherOpcode == Instr->Description.getOpcode())
       continue;
+    if (!State.getExegesisTarget().checkOpcodeSupported(
```

Comment:
Are these changes RISC-V specific? Could they be split into a separate patch?

Reply:
This is useful contribution, but it too small for a new patch, let's leave it here